package.json

```diff
@@ -54,9 +55,9 @@
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
     "ember-native-dom-event-dispatcher": "^0.6.3",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~2.18.0",
+    "ember-source": "~3.0.0-beta.5",
     "ember-source-channel-url": "^1.0.1",
-    "ember-try": "^0.2.23",
+    "ember-try": "^1.0.0-beta.1",
```

Comment:
offtopic: mmhhh.... I have thoughts about promoting this to 1.0.0...

Reply:
haha, we can always make a 2.0 😸 If there are specific things wrong with ember-try, please make issues so we can discuss the details.

IMHO, there is basically no reason to not have the ability to signal "new API's are added" (normal minor version bump) from "bug fixes" (normal patch release).

Reply:
https://github.com/ember-cli/ember-try/issues/155